django_project/lesson/templates/section/includes/section-list-detail.html

```diff
@@ -9,15 +9,17 @@ <h3>
     </div>
 
     <div class="col-lg-3">
-        <div class="btn-group pull-right">
-            <a class="btn btn-default btn-mini"
-               href='{% url "section-delete" project_slug=section.project.slug slug=section.slug %}'>
-                <span class="glyphicon glyphicon-minus"></span>
-            </a>
-            <a class="btn btn-default btn-mini"
-               href='{% url "section-update" project_slug=section.project.slug slug=section.slug %}'>
-                <span class="glyphicon glyphicon-pencil"></span>
-            </a>
-        </div>
+        {% if user.is_staff or user == project.owner %}
```

Comment:
we should have a role for lesson admin too - I think it was mentioned in the ticket....

Reply:
Yes, it's mentioned in the ticket. I'm aware of this for now but it's another task in our google doc. This PR was just for the CRUD of worksheet. Thanks for the reminder.